ui/less/containers.less

```diff
@@ -227,9 +227,12 @@
   div {
     font-size: 20px;
     line-height: 1.4;  // relative to font size.
-    background-color: rgba(0, 0, 0, 0.8);
     color: rgb(255, 255, 255);
   }
+
+  span {
+    background-color: rgba(0, 0, 0, 0.8);
```

Comment:
Suppose a region was to span half the video screen. Then if we left this line in the div, half the video screen would be a translucent black color. Even in general, it makes sense to set the background color property as a default only on the span element, since this is the element that contains the actual text.

Reply:
This is a problem because UITextDisplayer does not necessarily create spans at all.  If a cue contains no nested cues, UITextDisplayer only creates a div.  If that cue has no background, color, or font size attributes set, the defaults come from this CSS.

That is an implementation detail of UITextDisplayer, so it can change.  But as is, this CSS change will break things.

Reply:
Ah that makes sense. I was thinking about it from the perspective of every top-level cue (which corresponds to a div) will only be rendered if it contains nested cues (corresponding to spans) - so only the span elements which actually contain text should have a background color. I can revert for now since it would break stuff.